resources/js/processes-catalogue/components/ProcessChart.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div>
+    <div v-if="chart">
+      <div class="image-container">
+        <img
+          src="/img/launchpad-images/defaultImage.svg"
+          alt="Chart"
+          style="width: 100%; height: 100%; object-fit: cover"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { Line } from "vue-chartjs";
```

Comment:
vue-chartjs is not added in the package.json

Reply:
solved